receiver/awscontainerinsightreceiver/config_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awscontainerinsightreceiver
+
+import (
+	"path"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configtest"
+)
+
+func TestLoadConfig(t *testing.T) {
+	factories, err := componenttest.NopFactories()
+	assert.Nil(t, err)
+
+	factory := NewFactory()
+	receiverType := "awscontainerinsightreceiver"
+	factories.Receivers[config.Type(receiverType)] = factory
+	cfg, err := configtest.LoadConfigFile(
+		t, path.Join(".", "testdata", "config.yaml"), factories,
+	)
+
+	require.NoError(t, err)
+	require.NotNil(t, cfg)
+
+	assert.Equal(t, len(cfg.Receivers), 2)
+
+	//ensure default configurations are generated when users provide nothing
+	r0 := cfg.Receivers[receiverType]
+	assert.Equal(t, factory.CreateDefaultConfig(), r0)
+
+	r1 := cfg.Receivers["awscontainerinsightreceiver"]
+	assert.Equal(t, r1, factory.CreateDefaultConfig())
+
+	r2 := cfg.Receivers["awscontainerinsightreceiver/collection_interval_settings"].(*Config)
+	assert.Equal(t, r2,
+		&Config{
+			ReceiverSettings: config.ReceiverSettings{
+				TypeVal: config.Type(receiverType),
+				NameVal: "awscontainerinsightreceiver/collection_interval_settings",
+			},
```

Comment:
Please update core and this usage.

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/3321